TeamCode/src/main/java/org/firstinspires/ftc/teamcode/tuning/TuningOpModes.java

```diff
@@ -59,8 +59,8 @@ public static void register(OpModeManager manager) {
 
                 List<Encoder> leftEncs = new ArrayList<>(), rightEncs = new ArrayList<>();
                 List<Encoder> parEncs = new ArrayList<>(), perpEncs = new ArrayList<>();
-                if (md.localizer instanceof MecanumDrive.DriveLocalizer) {
-                    MecanumDrive.DriveLocalizer dl = (MecanumDrive.DriveLocalizer) md.localizer;
+                if (md.localizer instanceof MecanumDrive.MecanumDriveLocalizer) {
+                    MecanumDrive.MecanumDriveLocalizer dl = (MecanumDrive.MecanumDriveLocalizer) md.localizer;
```

Comment:
If these are subclasses of the drive class, why rename them?

Reply:
this was an artifact of my moving of the drive localizers out to separate files; could be changed